client/layout/popup-search/index.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/interactive-supports-focus */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize, useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import HelpSearchCard from 'calypso/blocks/inline-help/inline-help-search-card';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { getInlineHelpAdminSectionSearchResultsForQuery } from 'calypso/state/inline-help/selectors/get-inline-help-search-results-for-query';
+import hasInlineHelpAPIResults from 'calypso/state/selectors/has-inline-help-api-results';
+import { selectResult } from 'calypso/state/inline-help/actions';
+import { localizeUrl } from 'calypso/lib/i18n-utils';
+import QueryInlineHelpSearch from 'calypso/components/data/query-inline-help-search';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import getSearchQuery from 'calypso/state/inline-help/selectors/get-search-query';
+
+export function PopUpSearch( props ) {
+	const translate = useTranslate();
+
+	const onChildClick = ( e ) => e.stopPropagation();
+
+	const onResultClick = ( link ) => {
+		props.onClose();
+		window.location.href = localizeUrl( link );
+	};
+	return (
+		<div role="button" className="popup-search__mask" onClick={ props.onClose }>
+			<div className="popup-search__container" onClick={ onChildClick }>
+				<HelpSearchCard
+					forceFocus={ true }
```

Comment:
It looks like the `HelpSearchCard` component already has built-in force-focusing functionality: https://github.com/Automattic/wp-calypso/blob/trunk/client/blocks/inline-help/inline-help-search-card.jsx#L42 . From a quick test, I could achieve the same by removing `forceFocus={ true }` here and `location={ 'TEST' }` below. 

Is there a reason we can't just reuse its current behavior?